scripts/ci/build-installers.sh

```diff
@@ -48,8 +48,8 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$TARGET_ARCH" \
     -s "$(pwd)" \
-    -c 'make installers-all'
+    -c 'make electron-develop installers-all'
```

Comment:
You're welcome! Yeah, I think there are some ways to fix this. I'll see what I can do and hopefully send a PR soon.